db/schema.rb

```diff
@@ -19,8 +19,32 @@
     t.string "title"
   end
 
+  create_table "reviewings", force: :cascade do |t|
+    t.bigint "book_id"
+    t.bigint "reviews_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["book_id"], name: "index_reviewings_on_book_id"
+    t.index ["reviews_id"], name: "index_reviewings_on_reviews_id"
+  end
+
+  create_table "reviews", force: :cascade do |t|
+    t.integer "rating"
+    t.string "body"
+    t.bigint "books_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "users_id"
+    t.index ["books_id"], name: "index_reviews_on_books_id"
+    t.index ["users_id"], name: "index_reviews_on_users_id"
+  end
```

Comment:
This table is set up correctly as your join (foreign keys for books and users), but you need to use that relationship in your models and delete the `reviewings` table.